src/e2e-test/features/bigquerymultitablesink/MultipleDatabaseTableToBigQueryMultiTableSink_WithMacro.feature

```diff
@@ -0,0 +1,129 @@
+# Copyright © 2023 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+@BigQueryMultiTable_Sink
+Feature: BigQueryMultiTable sink -Verification of BigQuery to BigQueryMultiTable successful data transfer using macros
+
+  @MULTIPLEDATABASETABLE_SOURCE_TEST
+  Scenario:Verify data is getting transferred from BigQuery to BQMT sink with all datatypes using macros
```

Comment:
In the scenario it says we are running a pipeline from BQ to BQMT but we are selecting source as ```Multiple Database Tables``` which one is it ?

Reply:
Changed to Multiple Database Tables